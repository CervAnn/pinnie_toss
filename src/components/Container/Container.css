.Container {
  background-color: #5E6A68;
  height: 65vh;
  width: 100%;
  opacity: .8;
  display: flex;
  justify-content: center;
  align-items: center;
}

.buttonContainer {
  border: 2px solid black;
  height: 400px;
  width: 50%;
  margin: auto;
  display: flex;
  justify-content: space-evenly;
  align-items: center;
  border-radius: 15px;
}