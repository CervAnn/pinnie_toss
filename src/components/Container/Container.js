import React from 'react'
import '../Container/Container.css'
import PickALeague from '../PickALeague/PickALeague'

export const Container = () => {
  return (
    <section className="Container">
      <article className="buttonContainer">
        <PickALeague />
      </article>
    </section>
  )
}

export default Container
