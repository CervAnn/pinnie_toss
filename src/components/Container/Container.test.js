import React from 'react';
import { shallow } from 'enzyme';
import { Container } from '../Container/Container';

describe('Container', () => {
  it('should match the snapshot', () => {
    let wrapper = shallow(<Container />)
    expect(wrapper).toMatchSnapshot()
  })
})