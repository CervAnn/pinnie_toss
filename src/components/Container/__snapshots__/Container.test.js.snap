// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container should match the snapshot 1`] = `
<section
  className="Container"
>
  <article
    className="buttonContainer"
  >
    <Switch>
      <Route
        exact={true}
        path="/"
        render={[Function]}
      />
      <Route
        exact={true}
        path="/method"
        render={[Function]}
      />
      <Route
        exact={true}
        path="/randomPick"
        render={[Function]}
      />
    </Switch>
  </article>
</section>
`;
